The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/11132700904/job/30937160941) [(plain)](https://github.com/rust-lang/rust/commit/9be9141730ce8f78d5ffa9f72541f2eca13ab1d1/checks/30937160941/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain

COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---
#13 2.860 Building wheels for collected packages: reuse
#13 2.861   Building wheel for reuse (pyproject.toml): started
#13 3.119   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 3.120   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132715 sha256=dfa09868353292d98f811d3efdb0d54d07389e808efc71d68e3b93c514bf8bec
#13 3.120   Stored in directory: /tmp/pip-ephem-wheel-cache-xqtq8p2x/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#13 3.123 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#13 3.528 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#13 3.529 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 4.066 Collecting virtualenv
#13 4.066 Collecting virtualenv
#13 4.127   Downloading virtualenv-20.26.6-py3-none-any.whl (6.0 MB)
#13 4.388      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 6.0/6.0 MB 23.3 MB/s eta 0:00:00
#13 4.428 Collecting distlib<1,>=0.3.7
#13 4.437   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#13 4.451      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 37.0 MB/s eta 0:00:00
#13 4.484 Collecting platformdirs<5,>=3.9.1
#13 4.492   Downloading platformdirs-4.3.6-py3-none-any.whl (18 kB)
#13 4.527 Collecting filelock<4,>=3.12.2
#13 4.535   Downloading filelock-3.16.1-py3-none-any.whl (16 kB)
#13 4.619 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 4.809 Successfully installed distlib-0.3.8 filelock-3.16.1 platformdirs-4.3.6 virtualenv-20.26.6
#13 DONE 4.9s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      221120 kB
DirectMap2M:     8167424 kB
DirectMap1G:    10485760 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://static.rust-lang.org/dist/2024-09-22/rustfmt-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/2024-09-22/rustfmt-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt
---
##[endgroup]
fmt check
Diff in /checkout/compiler/rustc_data_structures/src/stack.rs:5:
 
 // Only the first stack that is pushed, grows exponentially (2^n * STACK_PER_RECURSION) from then
 // on. This flag has performance relevant characteristics. Don't set it too high.
-#[cfg(not(target_os="aix"))]
+#[cfg(not(target_os = "aix"))]
 const STACK_PER_RECURSION: usize = 1024 * 1024; // 1MB
 // LLVM for AIX doesn't feature TCO, increase recursion size for workaround.
-#[cfg(target_os="aix")]
+#[cfg(target_os = "aix")]
 const STACK_PER_RECURSION: usize = 16 * 1024 * 1024; // 16MB
 
 /// Grows the stack on demand to prevent stack overflow. Call this in strategic locations
fmt error: Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/compiler/rustc_ast_passes/src/feature_gate.rs" "/checkout/compiler/rustc_ast_passes/src/node_count.rs" "/checkout/compiler/rustc_ast_passes/src/errors.rs" "/checkout/compiler/rustc_ast_passes/src/show_span.rs" "/checkout/compiler/rustc_ast_passes/src/lib.rs" "/checkout/compiler/rustc_data_structures/src/graph/tests.rs" "/checkout/compiler/rustc_data_structures/src/graph/implementation/mod.rs" "/checkout/compiler/rustc_data_structures/src/graph/implementation/tests.rs" "/checkout/compiler/rustc_data_structures/src/graph/vec_graph/mod.rs" "/checkout/compiler/rustc_data_structures/src/graph/vec_graph/tests.rs" "/checkout/compiler/rustc_data_structures/src/graph/reference.rs" "/checkout/compiler/rustc_data_structures/src/graph/iterate/mod.rs" "/checkout/compiler/rustc_data_structures/src/graph/iterate/tests.rs" "/checkout/compiler/rustc_data_structures/src/stable_hasher.rs" "/checkout/compiler/rustc_data_structures/src/sync/freeze.rs" "/checkout/compiler/rustc_data_structures/src/sync/parallel.rs" "/checkout/compiler/rustc_data_structures/src/sync/vec.rs" "/checkout/compiler/rustc_data_structures/src/sync/lock.rs" "/checkout/compiler/rustc_data_structures/src/sync/worker_local.rs" "/checkout/compiler/rustc_data_structures/src/atomic_ref.rs" "/checkout/compiler/rustc_data_structures/src/obligation_forest/mod.rs" "/checkout/compiler/rustc_data_structures/src/obligation_forest/tests.rs" "/checkout/compiler/rustc_data_structures/src/obligation_forest/graphviz.rs" "/checkout/compiler/rustc_data_structures/src/tagged_ptr/copy/tests.rs" "/checkout/compiler/rustc_data_structures/src/tagged_ptr/impl_tag/tests.rs" "/checkout/compiler/rustc_data_structures/src/tagged_ptr/impl_tag.rs" "/checkout/compiler/rustc_data_structures/src/tagged_ptr/drop.rs" "/checkout/compiler/rustc_data_structures/src/tagged_ptr/copy.rs" "/checkout/compiler/rustc_data_structures/src/tagged_ptr/drop/tests.rs" "/checkout/compiler/rustc_data_structures/src/owned_slice.rs" "/checkout/compiler/rustc_data_structures/src/unhash.rs" "/checkout/compiler/rustc_data_structures/src/fingerprint.rs" "/checkout/compiler/rustc_data_structures/src/marker.rs" "/checkout/compiler/rustc_data_structures/src/frozen.rs" "/checkout/compiler/rustc_data_structures/src/base_n.rs" "/checkout/compiler/rustc_data_structures/src/unord.rs" "/checkout/compiler/rustc_data_structures/src/jobserver.rs" "/checkout/compiler/rustc_data_structures/src/lib.rs" "/checkout/compiler/rustc_data_structures/src/stack.rs" "/checkout/compiler/rustc_data_structures/src/sharded.rs" "/checkout/compiler/rustc_data_structures/src/sorted_map/index_map.rs" "/checkout/compiler/rustc_data_structures/src/sorted_map/tests.rs" "/checkout/compiler/rustc_data_structures/src/flat_map_in_place.rs" "/checkout/compiler/rustc_log/src/lib.rs" "/checkout/compiler/rustc_borrowck/src/prefixes.rs" "/checkout/compiler/rustc_borrowck/src/facts.rs" "/checkout/compiler/rustc_borrowck/src/dataflow.rs" "/checkout/compiler/rustc_borrowck/src/renumber.rs" "/checkout/compiler/rustc_borrowck/src/polonius/loan_invalidations.rs" "/checkout/compiler/rustc_borrowck/src/polonius/mod.rs" "/checkout/compiler/rustc_borrowck/src/polonius/loan_kills.rs" "/checkout/compiler/rustc_borrowck/src/path_utils.rs" "/checkout/compiler/rustc_borrowck/src/universal_regions.rs" "/checkout/compiler/rustc_borrowck/src/constraints/mod.rs" "/checkout/compiler/rustc_borrowck/src/constraints/graph.rs" "/checkout/compiler/rustc_borrowck/src/used_muts.rs" "/checkout/compiler/rustc_borrowck/src/consumers.rs" "/checkout/compiler/rustc_borrowck/src/nll.rs" "/checkout/compiler/rustc_borrowck/src/place_ext.rs" "/checkout/compiler/rustc_borrowck/src/util/mod.rs" "/checkout/compiler/rustc_borrowck/src/util/collect_writes.rs" "/checkout/compiler/rustc_borrowck/src/session_diagnostics.rs" "/checkout/compiler/rustc_borrowck/src/region_infer/opaque_types.rs" "/checkout/compiler/rustc_ast_passes/src/ast_validation.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Tue Oct  1 20:16:56 UTC 2024
  network time: Tue, 01 Oct 2024 20:16:56 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/compiler/rustc_data_structures/src/stack.rs
+++ b/compiler/rustc_data_structures/src/stack.rs
@@ -5,10 +5,10 @@ const RED_ZONE: usize = 100 * 1024; // 100k
 
 // Only the first stack that is pushed, grows exponentially (2^n * STACK_PER_RECURSION) from then
 // on. This flag has performance relevant characteristics. Don't set it too high.
-#[cfg(not(target_os="aix"))]
+#[cfg(not(target_os = "aix"))]
 const STACK_PER_RECURSION: usize = 1024 * 1024; // 1MB
 // LLVM for AIX doesn't feature TCO, increase recursion size for workaround.
-#[cfg(target_os="aix")]
+#[cfg(target_os = "aix")]
 const STACK_PER_RECURSION: usize = 16 * 1024 * 1024; // 16MB
 
 /// Grows the stack on demand to prevent stack overflow. Call this in strategic locations

